asg-tool/__main__.py

```diff
@@ -0,0 +1,47 @@
+#!/bin/env python3
+import argparse
+from asg_tool import get_spaces, bind_asg
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description="ASG tool to bind and check ASG's applied to CF spaces."
+    )
+
+    subparsers = parser.add_subparsers(
+        dest="command", help="Run <command> help for more information."
+    )
+
+    subparsers.add_parser("spaces-asg", help="List all spaces and their ASG's.")
```

Comment:
Meta: would it make sense to include an unbind command here? That way we could quickly roll back if we discover a problem?

Reply:
Totally. Bind, unbind and audit/check are three commands I'd imagine we'd need.